blog-posts/emails/article.md

```diff
@@ -0,0 +1,331 @@
+
+- Subscribe your devs email address to the topic to be warned in case of issue.
+- Subscribe a lambda automatically shutting down email sending in your configuration set.
+
+![Alarms schema](./assets/alarms.png 'Alarms schema')
+
+_Disabling a configuration set using Typescript and the AWS sdk v3:_
+
+```ts
+const client = new SESClient({});
+await client.send(
+  new UpdateConfigurationSetSendingEnabledCommand({
+    Enabled: false,
+    ConfigurationSetName: '<your_configuration_set_name>',
+  }),
+);
+```
+
+### Avoid sending emails to problematic recipients
+
+With these alarms, you already have a way to quickly react to issues concerning reputation. But you can do more to avoid sending multiple emails to addresses harming your reputation.
+
+One way to achieve this is to add bouncing or complaining addresses to a **suppression list**. Three types of suppression lists exist :
+
+- Global level (AWS as a whole)
+- Account level
+- Configuration set level
+
+Once again, I advise you to **setup a a configuration set level suppression list**.
+
+_Sample of configuration set config enabling suppression list of bounces and complaints:_
+
+```json
+{
+  "...": "...",
+  "SuppressionOptions": {
+    "SuppressedReasons": ["BOUNCE", "COMPLAINT"]
+  },
+  "...": "..."
+}
+```
+
+Great! You are now sure that your mails will be sent 🙃. But will your users receive them in their inbox? Probably not all of them if you don't customize your configuration a bit.
+
+## 2. Make sure you don't end up in your users spams
+
+When sending a user receives an email, there are many factors at stake, that will determine if the email ends up in the mailbox or in the spams.
+
+During my experience developing an email-sending service, I discovered [mail-tester][mail-tester]: a great tool to evaluate the probability of emails ending up in a inbox. Just send a test email to the address they give you, and it will analyze the likelihood of your message being well received.
+
+![Mail tester bad](./assets/mail-tester-bad.png 'Mail tester bad')
+
+This first mark is terrible, but you can fix it using the tips given my the website!
+
+Many factors determine if your email will be considered suspicious, let me cover the most important ones.
+
+### Email content
+
+The easiest to fix (it may already be OK on your app), but also the most important. Mail clients will often automatically classify your email as spam if you don't comply.
+
+- Your email should absolutely **have a subject and a body**.
+- Your email **shouldn't be to short**, especially if it contains images.
+- If your email contains links, prefer using **full links instead of minified ones**.
+- Also, check that every link included in your email is healthy and **redirects to a working website**.
+- Finally, a `List-Unsubscribe` header would improve your reputation, especially if you plan to send promotional emails. You will see in part 3 of this post how to implement it.
+
+### Sender domain
```

Comment:
This part feels a bit more confusing. There are multiple things to consider when it comes to sender reputation :

- SPF and IP sending pool. By default, SES uses a common pool of IPs, that have accumulated over time a quite bad reputation. Good practice is to have your own domain to be able to configure SPF to reference a private pool of IP configured on SES.
- DKIM - email signing. Providing proof that the MAIL FROM header is really the sender.
- DMARC - specify how to behave if MAIL FROM and sender are different, in order to avoid other people acting on your domain

I felt the explanation from Paul in the asakai (https://docs.google.com/presentation/d/1n3v_VBaEC5SRf3xwRUZSmQiYR8zoX9RVHOWzHtEMGhs/edit#slide=id.g1bb78460273_0_87 and https://drive.google.com/file/d/1Z3B5v1Ya55KRnkosurKvl-ZCzZk-2syX/view?usp=drivesdk) was really nit! Maybe you can andon him specifically on this part to make it more graspable (or take a few screenshots from his presentation to make the content clearer)